packages/mobile/src/harmony-native/components/Comments/Timestamp/Timestamp.tsx

```diff
@@ -0,0 +1,44 @@
+import { Text } from '../..'
+
+import type { TimeUnit, TimestampProps } from './types'
+import {
+  DAY_IN_MONTH,
+  HR_IN_DAY,
+  MIN_IN_HR,
+  MONTH_IN_YEAR,
+  MS_IN_S,
+  S_IN_MIN
+} from './types'
+
+const timeUnitMsMap: Record<TimeUnit, number> = {
+  m: MS_IN_S * S_IN_MIN,
+  h: MS_IN_S * S_IN_MIN * MIN_IN_HR,
+  d: MS_IN_S * S_IN_MIN * MIN_IN_HR * HR_IN_DAY,
+  mo: MS_IN_S * S_IN_MIN * MIN_IN_HR * HR_IN_DAY * DAY_IN_MONTH,
+  y: MS_IN_S * S_IN_MIN * MIN_IN_HR * HR_IN_DAY * DAY_IN_MONTH * MONTH_IN_YEAR
+} as const
+
+const getTextFromTime = (time: Date) => {
```

Comment:
nit: I think we should somehow include "largest unit" in the name, like `getLargestTimeUnitText`